pkg/cmd/server/api/v1/types.go

```diff
@@ -511,6 +511,14 @@ type ServingInfo struct {
 	ClientCA string `json:"clientCA"`
 	// NamedCertificates is a list of certificates to use to secure requests to specific hostnames
 	NamedCertificates []NamedCertificate `json:"namedCertificates"`
+	// MinTLSVersion is the minimum TLS version supported.
+	// TLS 1.0 = 0x0301
+	// TLS 1.1 = 0x0302
+	// TLS 1.2 = 0x0303
+	MinTLSVersion uint16 `json:"minTLSVersion,omitempty"`
+	// CipherSuites contains an overridden list of ciphers for the server to support.
+	// Values must match ciphers from http://www.iana.org/assignments/tls-parameters/tls-parameters.xml#tls-parameters-4
+	CipherSuites []uint16 `json:"cipherSuites,omitempty"`
```

Comment:
I think maintaining a mapping of human readable names to IANA versions across go versions will drift